SubmitReview/SubmitReview.ts

```diff
@@ -4,7 +4,7 @@ import { PullRequestAction } from '../lib/PullRequestAction';
 class SubmitReview extends PullRequestAction {
   
   protected async processJiraIssue(issueId: string): Promise<void> {
-    await this.moveIssue(issueId, "Request Changes");
+    await this.moveIssue(issueId, this.payload.review.state === 'changes_requested' ? 'Request Changes' : 'Approve');
```

Comment:
There's no `if` filtering for states in the YML, so this will trigger `Approve` transition when someone will do `Comment` on the PR. We need an `if` in the YML, so the action doesn't get scheduled and saves resources

Reply:
I have updated the yaml example in the `ReadMe.md`.
I couldn't find anything about adding an execution condition outside a workflow.

Reply:
..that's what I meant :)